test/testcafe/spec/EnrollAuthenticatorPhoneView_spec.js

```diff
@@ -144,6 +144,10 @@ test
     await challengePhonePageObject.waitForErrorBox();
     await t.expect(challengePhonePageObject.getInvalidOTPFieldError()).contains('Invalid code. Try again.');
     await t.expect(challengePhonePageObject.getInvalidOTPError()).contains('We found some errors.');
+    await t.wait(30500);
+    await t.expect(challengePhonePageObject.resendEmailView().hasClass('hide')).notOk();
+    const resendEmailView = challengePhonePageObject.resendEmailView();
+    await t.expect(resendEmailView.innerText).eql('Haven\'t received an SMS? Send again');
```

Comment:
Which test is for the form error clearing?

Reply:
There is no existing test for this.. Also the error messages clears and appears again because the submit will error out again.